CRAMv3.tex

```diff
@@ -1519,9 +1530,9 @@ \subsubsection*{CRAM index}
 \begin{enumerate}
 \item Sequence id
 
-\item Alignment start
+\item Alignment start (ignored for unmapped slices)
```

Comment:
require that it be set to some sentinel value here?  maybe 0 or -1

Reply:
Probably 0 if we wanted to pin it down.  -1 is ridiculously huge in CRAM anyway.

However practically speaking the main requirement is that it has no meaning and is ignored by the decoder.  If you do tie it down then alignment span should have the same action applied because they belong together.